README.md

```diff
@@ -20,6 +20,7 @@ Available addons
 addon | version | summary
```

Comment:
I can't see anything in there about formatting README files?

Also, given that this is a repository wide file, not module specific, I am a bit loathe to making changes beyond what is relevant to what I have changed.